CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 - **Breaking** Minimum supported Xcode version bumped to 12.2.
 
+## 1.45.2
```

Comment:
Please move your entry under `## Next` section. Do not add version by yourself. CI stamps changelog when release process is performing. :)

Reply:
@pepibumur @fortmarek Should we remove `## 1.46.0 - Momento` stamp from `CHANGELOG.md` since we removed `1.46.0` release?